test/test_file.py

```diff
@@ -0,0 +1,65 @@
+"""
+This module contains unit tests for file.py and related functions
+"""
+import os
+import pytest
+from pathlib import Path
+from test_sssd_conf import compare_conf_files
+from SCAutolib.models.sssd_conf import SSSDConf
+
+DIR_PATH = os.path.dirname(os.path.abspath(__file__))
+FILES_DIR = os.path.join(DIR_PATH, "files")
+
+
+def test_create_fail(file_test_prepare):
+    """
+    Exception is raised if conf file already exists
+    """
+    file_test = file_test_prepare
+    with file_test._conf_file.open('w') as config:
+        config.write('Test config file')
+    with pytest.raises(FileExistsError, match=f'{file_test._conf_file } already exists'):
+        file_test.create()
+
+
+def test_create(file_test_prepare):
+    """
+    Content of config file is created based on template if the file is not
+    present in the system.
+    """
+    file_test = file_test_prepare
+    file_test._template = SSSDConf._template
+    file_test.create()
+    # save content to file so it can easily be compared
+    file_test.save()
+    compare_conf_files(SSSDConf._template, file_test._conf_file)
```

Comment:
Hm, ConfigParser objects are represented as dictionaries. I think there is a better way to compare them without saving to the files